pkg/scheduler/api/types.go

```diff
@@ -114,6 +114,14 @@ type PriorityArgument struct {
 	LabelPreference *LabelPreference
 	// The RequestedToCapacityRatio priority function is parametrized with function shape.
 	RequestedToCapacityRatioArguments *RequestedToCapacityRatioArguments
+	// It uses a  resource name to identify the scarce resource
+	ResourceBinPacking *ResourceBinPacking
+}
+
+// ResourceBinPacking holds the resources that are used to configure the corresponding priority function
+type ResourceBinPacking struct {
+	// Used to identify resources to bin pack
+	Resource v1.ResourceName
```

Comment:
Would you help to give more explanation on what we're going to do for multiple resources? I'd like to avoid API changes in future :)

Reply:
@k82cn added explanation here https://docs.google.com/document/d/1LwOBNh554in0fjVFW5bKpVnG9nKNHBvO6r9hDCyyPik/edit#